src/common-types.ts

```diff
@@ -79,8 +79,8 @@ export interface CreatePdfResponse {
 export interface GetTweetsRequest {
   credentials?: firebase.auth.UserCredential;
   nextPageToken?: string;
-  fromDate: string; // yyyymmddhhmm format is expected here.
-  toDate: string; // yyyymmddhhmm format is expected here.
+  startDateTimeMs: number;
```

Comment:
Just to confirm; are these new fields also present for enterprise? I remember only seeing fromDate and toDate as options when I originally looked, but that was also a few years ago, so I'm not up to date!

Reply:
Got it, thanks!